cpp/CMakeLists.txt

```
@@ -763,6 +763,22 @@ if(HT_DEFAULT_ALLOCATOR)
     set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -DHT_DEFAULT_ALLOCATOR")
 endif(HT_DEFAULT_ALLOCATOR)
 
+# Set a default logging level if none was specified
+set(DEFAULT_LOGGING_LEVEL OFF)
```

In rapidsai/rmm#556 I changed this to INFO, because in that PR I also made it so by default there are no "Info" messages unless there are higher-level messages (warnings / errors). The header is "info", but it only gets written the first time the logger is used.

So I think we should match RMM here:

```suggestion
set(DEFAULT_LOGGING_LEVEL INFO)
```

If we don't, then in shipped binaries there will be no log created when errors are hit, which is a nice feature to have...

---

I originally set it to `OFF` to fix the problem of RMM always emitting `rmm_log.txt` files.  I agree `INFO` is a more appropriate setting once RMM's default behavior with `rmm_log.txt` is fixed.